MonoGame.Framework/BoundingSphere.cs

```diff
@@ -4,55 +4,73 @@
 
 using System;
 using System.Collections.Generic;
-using System.Globalization;
-using System.Runtime.Serialization;
 using System.Diagnostics;
+using System.Runtime.Serialization;
 
 namespace Microsoft.Xna.Framework
 {
-
+    /// <summary>
+    /// Describes a virtual sphere in 3D-space for bounding operations.
+    /// </summary>
     [DataContract]
     [DebuggerDisplay("{DebugDisplayString,nq}")]
     public struct BoundingSphere : IEquatable<BoundingSphere>
     {
         #region Public Fields
 
+        /// <summary>
+        /// The sphere center.
+        /// </summary>
         [DataMember]
         public Vector3 Center;
 
+        /// <summary>
+        /// The sphere radius.
+        /// </summary>
         [DataMember]
         public float Radius;
 
-        #endregion Public Fields
+        #endregion
+
+        #region Internal Properties
+
+        internal string DebugDisplayString
+        {
+            get
+            {
+                return string.Concat(
+                    "Pos( ", this.Center.DebugDisplayString, " )  \r\n",

```

Comment:
I think Pos in DebugDisplayString is incorrect and should be changed to Center... what do you think @tomspilman, @KonajuGames ?